darts/utils/statistics.py

```diff
@@ -255,6 +255,56 @@ def remove_trend(ts: TimeSeries,
     return new_ts
 
 
+def granger_causality_tests(ts_cause: TimeSeries,
+                            ts_effect: TimeSeries,
+                            maxlag: int,
+                            addconst: bool = True,
+                            verbose: bool = True
+                          ) -> None:
+    
+    """
+    Provides four tests for granger non causality of 2 time series using `statsmodels.tsa.stattools.grangercausalitytests`.
+
+
+    Parameters
+    ----------
+    ts_cause
+        An univariate time series. The statistical test determines if this time series 
+        'Granger causes' the time series ts_effect (second parameter). Missing values are not supported.
+        if H_0 is (non causality) is rejected (p near 0), then there is a 'granger causality'.
+    ts_effect
+        Univariate time series 'Granger caused' by ts_cause.
+    maxlag
+        If an integer, computes the test for all lags up to maxlag. 
+        If an iterable, computes the tests only for the lags in maxlag.
+    addconst
+        Include a constant in the model.
+    verbose
+        Print results.
+    Returns
+    -------
+    Dict
```

Comment:
Could we maybe link to statsmodels doc here - as we return exactly what they return? (We can also leave the current explanations in our docstring too).

Reply:
I slightly changed the explanation, but the parameter doc is the same. In case of link to statsmodels doc, just a link is enough in introduction?

Reply:
Yes, i think it's enough.